src/transport_udp.cpp

```diff
@@ -524,8 +553,27 @@ UDPTransport::connect_client()
   // Notify caller that the connection is now ready
   delegate.on_connection_status(last_context_id, TransportStatus::Ready);
 
+  #if defined(PLATFORM_ESP)
+  auto cfg = create_config("FDReader", 1, 12 * 1024, 5);
```

Comment:
Maybe add transport config options?

Reply:
good idea. but these are test parameters to verify the functionality. Once we have the builds integrated and can test the options, happy to move it as config option